app/controllers/api/users_controller.rb

```diff
@@ -0,0 +1,95 @@
+class Api::UsersController < Api::ApiController
+
+    before_action :authenticate_with_http_token
+
+    before_action :set_user, only: [:show, :edit, :update, :destroy]
+
+    # GET /users or /users.json
+    def index
+        @users = User.all
+        render json: { users: @users }, status: :ok
+    end
```

Comment:
Why are you retrieving a list of users if this action is not accessible through your routes then it should not be listed here